File: python/small-enough.py
# Tanggal: 11 Mei 2026
# Link: https://www.codewars.com/kata/57cc981a58da9e302a000214/train/python

def small_enough(array, limit):
    for num in array:
        if num > limit:
            return False
    return True

if __name__ == "__main__":
    print(small_enough([66, 101], 200))
    print(small_enough([66, 201], 200))

File: js/small-enough.js
/**
 * Tanggal: 10 September 2026
 * Link: https://www.codewars.com/kata/542c0f198e077084c0000c2e/train/javascript
 */

function smallEnough(array, limit) {
    for (const num of array) {
        if (num > limit) {
            return false;
        }
    }

    return true;
}

console.log(smallEnough([66, 101], 200));
